.gitmodules

```diff
@@ -7,4 +7,3 @@
 [submodule "doc/CMakeModules"]
   path = doc/CMakeModules
   url = https://github.com/noaa-emc/cmakemodules
-  branch = develop
```

Comment:
This needs to be reverted

Reply:
i never touched this file. very odd. in any case, reverted.